<ui version="4.0" >
 <class>Form1</class>
 <widget class="QDialog" name="Form1" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Wikipedia dump reader</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="6" column="2" >
    <widget class="QPushButton" name="pushButton2" >
     <property name="text" >
      <string>apply python regex</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QCheckBox" name="checkBox1" >
     <property name="text" >
      <string>debug mode</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>textLabel1</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="margin" >
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>101</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton9" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QTextEdit" name="textEdit2" />
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="QTextEdit" name="textEdit3" >
    </widget>
   </item>
   <item rowspan="5" row="2" column="3" >
    <widget class="QListWidget" name="listBox3" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>4</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton4" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>28</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit1" />
     </item>
     <item>
      <widget class="QPushButton" name="pushButton1" >
       <property name="text" >
        <string>Go</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton3" >
       <property name="text" >
        <string>search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox3" >
       <property name="text" >
        <string>searchbox</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QListWidget" name="listBox1" >
     <property name="maximumSize" >
      <size>
       <width>32767</width>
       <height>110</height>
      </size>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QTextBrowser2" name="textEdit1" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QTextBrowser2</class>
   <extends>QTextBrowser</extends>
   <header>QTextBrowser2</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton4</tabstop>
  <tabstop>lineEdit1</tabstop>
  <tabstop>pushButton1</tabstop>
  <tabstop>pushButton3</tabstop>
  <tabstop>listBox1</tabstop>
  <tabstop>checkBox3</tabstop>
  <tabstop>textEdit1</tabstop>
  <tabstop>pushButton9</tabstop>
  <tabstop>listBox3</tabstop>
  <tabstop>checkBox1</tabstop>
  <tabstop>textEdit2</tabstop>
  <tabstop>textEdit3</tabstop>
  <tabstop>pushButton2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox1</sender>
   <signal>toggled(bool)</signal>
   <receiver>textEdit3</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>29</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel" >
     <x>33</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox1</sender>
   <signal>toggled(bool)</signal>
   <receiver>textEdit2</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>29</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel" >
     <x>33</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton4</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>113</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox3</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton3</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>578</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel" >
     <x>497</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox3</sender>
   <signal>toggled(bool)</signal>
   <receiver>listBox1</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>578</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel" >
     <x>31</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox1</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton2</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>56</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel" >
     <x>482</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
